source/client-side-operations-timeout/tests/override-operation-timeoutMS.yml

```diff
@@ -0,0 +1,1920 @@
+# Tests in this file are generated from override-operation-timeoutMS.yml.template.
+
+description: "timeoutMS can be overriden for an operation"
```

Comment:
I don't think we'll run any of the tests in this file in the Go driver. Users can pass a `Context` with a deadline to any operation to specify an operation-level timeout. Allowing users to set `timeoutMS` on an operation _and_ pass a `Context` with a deadline would be redundant and confusing.

Reply:
On an unrelated note, I realized there's a typo in the test name: overridden

Reply:
What do you think about running these tests by having the test runner translate the `timeoutMS` argument into a Context with an equivalent deadline. This is what I've done in Python. And I've fixed the typo.

Reply:
This might require more discussion in the unified spec, but would the proposal be that drivers implementing `timeoutMS` via a context modify their execution such that each operation with a `timeoutMS` option runs within its own context?

Reply:
Yes, but I think the description of that behavior should live in the CSOT test readme, not the unified spec.

Reply:
I can try to do that in Go and report back. @jmikola yep that's my understanding of the proposal.

Reply:
> but I think the description of that behavior should live in the CSOT test readme, not the unified spec.

No objections to hosting that documentation in the CSOT spec, but I think it'd be prudent to reference the CSOT spec's explanation from [Executing an Operation](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#executing-an-operation) in the unified spec. In the event `timeoutMS` ever shows up in non-CSOT spec tests, it'd be helpful to have this mentioned _somewhere_ in the test runner implementation directions.

Reply:
My idea doesn't work (at least for Go) so drivers won't be using this approach. Given that I don't think there's anything special to document in the unified format regarding the timeoutMS option.